src/test/java/wedlog/address/logic/parser/CommandParserTestUtil.java

```diff
@@ -20,7 +20,7 @@ public static void assertParseSuccess(Parser<? extends Command> parser, String u
             Command command = parser.parse(userInput);
             assertEquals(expectedCommand, command);
         } catch (ParseException pe) {
-            throw new IllegalArgumentException("Invalid userInput.", pe);
+            throw new IllegalArgumentException("Invalid userInput: " + pe.getMessage(), pe);
```

Comment:
When this error is thrown, this change means that the error message is shown twice.